userguide/getting-started/quick_start_with_kb_api.adoc

```diff
@@ -628,3 +650,6 @@ $sub = $subscriptionApi->createSubscription(
   </div>
 </div>
 ++++
+
+[NOTE]
+*Note:* You can view the invoice that is automatically generated by going to the account in Kaui and clicking the Invoices sub-menu. For more information, see https://docs.killbill.io/latest/quick_start_with_kaui.html#step6[View the Generated Invoice] in the _Quick Start with Kaui_ guide. 
```

Comment:
Again, instead of pointing to Kaui, I would suggest pointing to the API endpoints in the slate docs: [Retrieve account invoices](https://killbill.github.io/slate/#account-retrieve-account-invoices) and [Retrieve invoice by id](https://killbill.github.io/slate/#invoice-retrieve-an-invoice-by-id).

Reply:
DONE